.pylintrc

```diff
@@ -76,6 +76,7 @@ disable=invalid-name,
         fixme,  # We deliberately use TODO/FIXME inline
         abstract-class-instantiated,  # Needed for can.Bus
         duplicate-code,  # Needed due to https://github.com/PyCQA/pylint/issues/214
+        keyword-arg-before-vararg,
```

Comment:
I am a bit hesitant to adding new excepts, especially since this specific earning has caused problems in the past. I get these warnings locally:

```
************* Module can.io.generic
can/io/generic.py:31:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
can/io/generic.py:92:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
************* Module can.io.canutils
can/io/canutils.py:135:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
************* Module can.io.csv
can/io/csv.py:94:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
************* Module can.io.printer
can/io/printer.py:28:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
************* Module can.io.sqlite
can/io/sqlite.py:35:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
can/io/sqlite.py:137:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
************* Module can.io.logger
can/io/logger.py:320:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
************* Module can.io.asc
can/io/asc.py:37:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
can/io/asc.py:352:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
************* Module can.io.blf
can/io/blf.py:372:4: W1113: Keyword argument before variable positional arguments list in the definition of __init__ function (keyword-arg-before-vararg)
```

I think we should rather change these IO handlers.

Reply:
I agree, will do that.